dist/PageBanner.css

```diff
@@ -1,11 +1,32 @@
-.page-banner {
+.page-banner-container {
```

Comment:
the container is another element - so if we follow BEM guidelines it should be liek:
`page-banner__container`